packages/terra-progress-bar/src/ProgressBar.jsx

```diff
@@ -31,36 +27,36 @@ const propTypes = {
    * Sets the maximum possible fill-value.
    */
   max: PropTypes.number,
-  /**
-   * Boolean check for gradient.
-   */
-  hasGradient: PropTypes.bool,
   /**
    * Value passed to aria-valuetext for accessibility. You can view more about this attribute
    * at https://www.w3.org/WAI/PF/aria/states_and_properties#aria-valuetext.
    */
   valueText: PropTypes.string,
+  /**
+   * Sets an author defined class, to control the colors of the progress bar
```

Comment:
```suggestion
   * Sets an author defined class, to control the colors of the progress bar.
   * 
   * ![IMPORTANT](https://badgen.net/badge//IMPORTANT/blue?icon=github)
   * Adding `--my-app...` CSS variables is required for proper re-themeability when creating your custom color styles _(see included examples)_.
```
\
\
Adding reminder to use CSS variables:

![IMPORTANT](https://badgen.net/badge//IMPORTANT/blue?icon=github) Adding `--my-app...` CSS variables is required for proper re-themeability when creating your custom color styles _(see included examples)_.